src/SFA.DAS.AssessorService.Application.Api.UnitTests/Controllers/Register/Command/UpdateOrganisationNotFoundTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Threading;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Api.Types.Models.AO;
+using SFA.DAS.AssessorService.Api.Types.Models.Register;
+using SFA.DAS.AssessorService.Application.Api.Controllers;
+using SFA.DAS.AssessorService.Application.Exceptions;
+using SFA.DAS.AssessorService.Data.DapperTypeHandlers;
+using SFA.DAS.AssessorService.Domain.Consts;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Controllers.Register.Command
+{
+    [TestFixture]
+    public class UpdateOrganisationNotFoundTests
+    {
+        private static RegisterController _controller;
+        private static Mock<IMediator> _mediator;
+        private static Mock<ILogger<RegisterController>> _logger;
+        private object _result;
+        private EpaOrganisation _returnedOrganisation;
```

Comment:
_returnedOrganisation isn't being used. Are you missing a test?

Reply:
I is a neglected leftover when the original spec had the organisation being returned with the update, rather than an OK and an organisation ID. I've removed this now.  Eagle Eyes!